src/sentry/integrations/slack/message_builder/incidents.py

```diff
@@ -40,20 +40,17 @@ def __init__(
         self.new_status = new_status
         self.chart_url = chart_url
 
-    def build(self) -> SlackBody:
+    def build(self) -> Tuple[SlackBody, str]:
```

Comment:
I'm not super happy about returning a tuple, but it felt like the best way to get the fallback text out of this function.